rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -0,0 +1,255 @@
+from rotkehlchen.utils.misc import (
+    hex_or_bytes_to_address,
+    hex_or_bytes_to_int,
+    hex_or_bytes_to_str,
+    ts_sec_to_ms,
+)
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.chain.evm.node_inquirer import EvmNodeInquirer
+    from rotkehlchen.user_messages import MessagesAggregator
+
+from ..constants import CPT_AAVE_V2
+
+ENABLE_COLLATERAL = b'\x00\x05\x8aV\xea\x94e<\xdfO\x15-"z\xce"\xd4\xc0\n\xd9\x9e*C\xf5\x8c\xb7\xd9\xe3\xfe\xb2\x95\xf2'  # noqa: E501
+DISABLE_COLLATERAL = b'D\xc5\x8d\x816[f\xddK\x1a\x7f6\xc2Z\xa9{\x8cq\xc3a\xeeI7\xad\xc1\xa0\x00\x00"}\xb5\xdd'  # noqa: E501
+DEPOSIT = b'\xdehW!\x95D\xbb[wF\xf4\x8e\xd3\x0b\xe68o\xef\xc6\x1b/\x86L\xac\xf5Y\x89;\xf5\x0f\xd9Q'
+WITHDRAW = b'1\x15\xd1D\x9a{s,\x98l\xba\x18$N\x89zE\x0fa\xe1\xbb\x8dX\x9c\xd2\xe6\x9el\x89$\xf9\xf7'  # noqa: E501
+BORROW = b'\xc6\xa8\x980\x9e\x82>\xe5\x0b\xacd\xe4\\\xa8\xad\xbaf\x90\xe9\x9exA\xc4]uN*8\xe9\x01\x9d\x9b'  # noqa: E501
+REPAY = b'L\xdd\xe6\xe0\x9b\xb7U\xc9\xa5X\x9e\xba\xecd\x0b\xbf\xed\xff\x13b\xd4\xb2U\xeb\xf83\x97\x82\xb9\x94/\xaa'  # noqa: E501
+
+
+class Aavev2Decoder(DecoderInterface):
+    def __init__(
+            self,
+            evm_inquirer: 'EvmNodeInquirer',
+            base_tools: 'BaseDecoderTools',
+            msg_aggregator: 'MessagesAggregator',
+    ) -> None:
+        super().__init__(evm_inquirer, base_tools, msg_aggregator)
+        self.evm_inquirer = evm_inquirer
+
+    def _decode_collateral_events(
+            self,
+            token: 'EvmToken',
+            transaction: EvmTransaction,
+            tx_log: EvmTxReceiptLog,
+    ) -> HistoryBaseEntry:
+        user = hex_or_bytes_to_address(tx_log.topics[2])
+        return HistoryBaseEntry(
+            event_identifier=transaction.tx_hash,
+            sequence_index=tx_log.log_index,
+            timestamp=ts_sec_to_ms(transaction.timestamp),
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=token,
+            balance=Balance(),
+            notes=f'{"Enable" if tx_log.topics[0] == ENABLE_COLLATERAL else "Disable"} {token.symbol} ({token.name}) as collateral on AAVE v2 for user {user}',  # noqa: E501
+            counterparty=CPT_AAVE_V2,
+        )
+
+    def _decode_deposit(
+            self,
+            token: 'EvmToken',
+            tx_log: EvmTxReceiptLog,
+            decoded_events: list[HistoryBaseEntry],
+    ) -> None:
+        user = hex_or_bytes_to_address(tx_log.data[:32])
+        on_behalf_of = hex_or_bytes_to_address(tx_log.data[64:96]) if hex_or_bytes_to_str(tx_log.data[64:96]) != '' else None  # noqa: E501
+        amount = asset_normalized_value(
+            amount=hex_or_bytes_to_int(tx_log.data[32:64]),
+            asset=token,
+        )
+        notes = f'Deposit {amount} {token.symbol} ({token.name}) from user {user} into AAVE v2'
```

Comment:
I mentioned the (token.name) but because you were doing something strange or ommiting it. I would just leave `amount token.symbol` that is what we do in other decoders and in the case of aave is clear. My fault if it was a misunderstanding, sorry

Reply:
Alright :ok_hand: